environments/autostart.md

```diff
@@ -6,26 +6,20 @@ description: Learn how to configure automated environment rebuilds.
 Coder [automatically turns off idle
 environments](../admin/environment-management/shutdown.md) to help manage
 resource expenditure. Typically, this means environments turn off overnight and
-remain offline until a rebuild is requested. With Autostart, rebuilds may be
-automatically requested at a time that best suits your workflow. You can expect
-your environments to be ready for you at the start of each work day.
+remain offline until a rebuild is requested. With Autostart, you can request
+rebuilds at a time that suits your workflow. You can expect your environments to
+be ready for you at the start of each workday.
 
 ## Criteria for Autostart
 
-Your environment must meet the following criteria for you to use Autostart:
+Your environment must be
 
-1. You've [set the Autostart time and selected
-   environments](#enabling-autostart) to be started automatically at that time.
+- Active (e.g., has been opened within the last five days)
```

Comment:
If it hasn't been opened in 5 days, it won't trigger. 

Inactive means 5 days or more.

With that said, active would mean it's been opened in the last 4 days.